e2e/ci-e2e-config.yaml

```diff
@@ -0,0 +1,68 @@
+# This file contains configuration for running e2e tests.
+# Many of these fields can be overridden with environment variables.
+# All fields that support this have the corresponding environment variable name in a comment beside the field.
+
+
+# | Environment Variable | Description                               | Default Value                |
+# |----------------------|-------------------------------------------|------------------------------|
+# | CHAIN_IMAGE          | The image that will be used for the chain | ghcr.io/cosmos/ibc-go-simd   |
+# | CHAIN_A_TAG          | The tag used for chain A                  | N/A (must be set)            |
+# | CHAIN_B_TAG          | The tag used for chain B                  | N/A (must be set)            |
+# | CHAIN_BINARY         | The binary used in the container          | simd                         |
+# | RELAYER_TAG          | The tag used for the relayer              | 1.10.0                       |
+# | RELAYER_ID           | The type of relayer to use (rly/hermes)   | hermes                       |
+
+
+# see sample.config.yaml for a bare minimum configuration example.
+# set env E2E_CONFIG_PATH to point to this file to use it.
+---
+chains:
+  # the entry at index 0 corresponds to CHAIN_A
+  - chainId: chainA-1
+    numValidators: 4
+    numFullNodes: 1
+    image: ghcr.io/cosmos/ibc-go-simd # override with CHAIN_IMAGE
+    binary: simd # override with CHAIN_BINARY
+
+    # the entry at index 1 corresponds to CHAIN_B
+  - chainId: chainB-1
+    numValidators: 4
+    numFullNodes: 1
+    image: ghcr.io/cosmos/ibc-go-simd # override with CHAIN_IMAGE
+    binary: simd # override with CHAIN_BINARY
+
+activeRelayer: hermes # override with RELAYER_ID
```

Comment:
I assume this has to be one of the IDs in the list below, maybe worth specifying it in the comment?